english/beach/beach.md

```diff
@@ -0,0 +1,8 @@
+You know in your heart the beach is where you wanna be.
+The sun, the sand, the sea and the breeze.
+You close your eyes and you can just imagine the beauty. Without any hesitation you head for the door.
+
+Do you:
+[Call your friends?](https://github.com/jonotko/create-your-own-adventure/blob/jono-adventure/english/go-with-friends/go-with-friends.md)
```

Comment:
Please don't use absolute links - these should be relative links instead. That way, whether you are in the original repository or a fork, you will always stay within that same repository when clicking links.